Documentation/ceph-osd-mgmt.md

```diff
@@ -66,7 +66,7 @@ and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly c
 ### From the Toolbox
 
 1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`.
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you set the corresponding deployment replicas to 0 beforehand.
```

Comment:
Can you add the command for the user to do so? So the doc has everything

Reply:
Added, thanks.